dashboard/dashapi/dashapi.go

```diff
@@ -333,6 +334,8 @@ type BugReport struct {
 	ReproCLink        string
 	ReproSyz          []byte
 	ReproSyzLink      string
+	MachineInfo       []byte
+	MachineInfoLink   string
```

Comment:
I added these fields but I am not sure how these APIs are used externally and if doing this will break them or not.

Reply:
If you are just adding fields, it should not break anybody.